addons/mail/static/src/core/thread_model.js

```diff
@@ -394,4 +394,12 @@ export class Thread {
     getMemberName(persona) {
         return persona.name;
     }
+
+    getPreviousMessage(message) {
+        const previousNonEmptyMessages = this.nonEmptyMessages.filter(({ id }) => id < message.id);
+        if (previousNonEmptyMessages.length === 0) {
+            return false;
+        }
+        return this._store.messages[Math.max(...previousNonEmptyMessages.map((m) => m.id))];
```

Comment:
Not a big deal, but naming variables in a simpler way makes code more readable.
Here just `previous` is good.